eclair-core/src/test/scala/fr/acinq/eclair/io/PeerSpec.scala

```diff
@@ -237,6 +242,22 @@ class PeerSpec extends TestkitBaseClass {
     probe.expectMsg("disconnecting")
   }
 
+  test("use correct fee rates when spawning a channel") { f =>
+    import f._
+
+    val probe = TestProbe()
+    system.eventStream.subscribe(probe.ref, classOf[ChannelCreated])
+    connect(remoteNodeId, authenticator, watcher, router, relayer, connection, transport, peer)
+
+    assert(peer.stateData.channels.isEmpty)
+    probe.send(peer, Peer.OpenChannel(remoteNodeId, Satoshi(12300), MilliSatoshi(0), None, None, None))
+    awaitCond(peer.stateData.channels.nonEmpty)
+
+    val channelCreated = probe.expectMsgType[ChannelCreated]
+    assert(channelCreated.initialFeeratePerKw == feeEstimator.getFeeratePerKw(Alice.nodeParams.feeTargets.commitmentBlockTarget))
```

Comment:
No need to change the fixture, and also there shouldn't be calls to `Alice.nodeParams` because the peer is only basing its parameters on those.

```scala
val nodeParams = peer.underlyingActor.nodeParams
assert(channelCreated.initialFeeratePerKw == nodeParams.feeEstimator.getFeeratePerKw(nodeParams.feeTargets.commitmentBlockTarget))
assert(channelCreated.fundingTxFeeratePerKw.get == nodeParams.feeEstimator.getFeeratePerKw(nodeParams.feeTargets.fundingBlockTarget))
```

Reply:
Also done at 8856051